### PR TITLE
ion carbine - stopgap between pistol and rifle

### PR DESCRIPTION
**Pull request by SonixApache**
Made as a 'better' alternative to #22765, since I don't like just jamming current-age shit in the protolathe.
Basically a higher capacity ion pistol that doesn't fit in holsters / can't be dualwielded.
Personally, I think EMP guns shouldn't be that easily available at all, but if we're going to have the newfigs :+1:creep it, it might as well be something unique.

If this doesn't pass, I'd like to just remove it from the lathe.

comparison  inhands 

🆑 
 - rscadd: Added the Ion Carbine, a smaller ion rifle. Available from R&D.

### CODE REVIEW

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
So you can now print Ion Ri🅱️le (BUT FITS IN YOUR BACKPACK) instead of just ion rifles

**Comment by SonixApache** [comment]
if you think it should be large sized that wouldn't be too bad an idea

**Comment by brndd** [comment]
\*👍creeps it\*

but yeah maybe make it large sized @SonixApache. I don't think ion rifles are suddenly going to become meta considering the overpowered ion boogeyman barely ever manifests itself as is despite every major player (sec, ERT, syndies, voxraiders...) having access to them, but making it large sized is at least something of a failsafe for that contingency.

**Comment by Irradiation** [comment]
If it doesn't fit in backpacks, then what's the difference between it and normal Ion Guns? ~~Cargo~~ R&D doesn't have to be niggers about tsci'ing Ions out of the armory?

File: code/game/gamemodes/wizard/rightandwrong.dm [hunk 1]
```diff
@@ -142,8 +142,10 @@
 			new /obj/item/weapon/gun/projectile/colt(get_turf(src))
 		if("lawgiver")
 			new /obj/item/weapon/gun/lawgiver(get_turf(src))
-		if("ion")
-			new /obj/item/weapon/gun/energy/ionrifle/ionpistol(get_turf(src))
+		if("ionpistol")
+			new /obj/item/weapon/gun/energy/ionrifle/ioncarbine(get_turf(src))
+		if("ioncarbine")
+			new /obj/item/weapon/gun/energy/ionrifle/ioncarbine/ionpistol(get_turf(src))
```
Comment (SonixApache) [inline code/game/gamemodes/wizard/rightandwrong.dm:146]: 🤔

Comment (Irradiation) [reply code/game/gamemodes/wizard/rightandwrong.dm]: nice one

**Comment by brndd** [comment]
> If it doesn't fit in backpacks, then what's the difference between it and normal Ion Guns? ~Cargo~ R&D doesn't have to be niggers about tsci'ing Ions out of the armory?

Literally just something to placate people who think it'd be overpowered.

**Comment by SonixApache** [comment]
the only issue i have with making it large-sized is that i'd have to make on-back sprites
and i really fucking hate spritework
also it doesn't have a sling mount -- our ion rifle doesn't have a sling either but at least it's long enough

As for the *difference*, this one fires weaker ion bolts and has less capacity.

**Comment by Irradiation** [comment]
People seldom even touch Ion Pistols anyway (BECAUSE PACKED ONLY), so for the sake of experimenting, I'd say try making them fit in backpacks and see what happens, tbh. If they end up being ion-gun-but-different-name-that-can-be-printed, I doubt it'll get used as R&D barely prints guns nowadays and people seldom even use the default ion guns in the armory.

Alter-alternatively, try putting ion pistols in box armory and see if people bother to get those.

**Comment by brndd** [comment]
Perhaps making it large sized is something to be done if these actually turn out to be as overpowered as some fear. I think they should fire regular-sized projectiles though; the small ones only deal full damage on direct hits whereas the large ones deal full damage in a 1 tile radius (ie. a 3x3 area). The half-damage radius is the same for both (3x3), so this only makes these weaker against mechs and whatever else may be able to deflect ion bolts, really, and mechs are already strong enough against ions IMO.

**Comment by Irradiation** [comment]
I honestly think that "people" are theorycrafting a bit too much about the potential power of a Protolathe gun :man_shrugging:

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Leaving the "ions in protolathe" discussion aside (that was mostly in #22765)
Being able to fit in your backpack is a huge selling point, especially for something you can print multiples of easily and just carry 2-3 spares in your backpack
So in my eyes this PR is identical to #22765 except _more_ creepy, not less

**Comment by Irradiation** [comment]
I unironically would like to see what would happen if you replaced the Ion Guns in the armory with their Pistol variants.

**Comment by aceedex** [comment]
👎  because powercreep
But the thing looks nice so I like it.
As always my recommended solution is to make the Ion rifles free of the MECH_SCAN_FAIL check
as to give both more unique things to scan and prevent dude 20 ion lmao

**Comment by brndd** [comment]
If ions are so powerful why are they in the armoury at roundend in 90% of rounds?

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Them being in the armory probably has a lot to do with that

**Comment by brndd** [comment]
> Them being in the armory probably has a lot to do with that

It's not difficult to break into the armoury, and if they were really that good then at least sec would pick them up. Newcops also get them and never ever pick them up because guess what they aren't actually overpowered.

**Comment by SonixApache** [comment]
- They're only available from the secure armory
 - They're fuckhuge and not too easy to conceal while keeping them on your person
 - Sec doesn't need them against the average threat
 - Why would you need an ion gun if you intent to just stun and instakill everyone or not even shoot anyone in the first place

weak arguments tbh

**Comment by DrSnips** [comment]
I agree with the statement based on nuke ops. They just don't have the stopping power in practice like everyone theorizes. As for matter of size I would just go with the small/tiny, medium, and large like we have for all other of the energy arms. We don't seem to have any issue with those currently so if it's not broken I don't see a reason to "fix" it.

**Comment by brndd** [comment]
I mean, they have NO stopping power. They don't do shit to organics unless they have artificial organs (which almost nobody does for obvious reasons). Their combat advantage is in draining your guns and disabling headsets, both of which are countered by just running away and coming back with ballistic weapons, or running away to recharge and coming back when your enemy is out of charge in their rifle.

I don't think it's worthwhile to consider the ifs and buts of the matter, ions are not widely used now and I don't think putting them in R&D is going to change that, and if it does that bridge is easily crossed once we get to it.

**Comment by aceedex** [comment]
I personally always take a ion rifle as an op. Player behavior and competence isnt exactly a great argument for the strength of an item. What an Ion does security has access to/simply does not need except when particularly fighting silicons. If security executes someone they usually do under the assumption that the person was OK to be executed, lest admins intervene, and so it is not 100% necessary for them to care about disabling communications. To open doors a warden has a battering ram or simply the help of an AI or heads.  most traitor do NOT carry empable weapons and even if they do an EMP isnt a 100% disarm. that is why you see them in the armory 90% of rounds. I say its a good thing that we currently dont have a way for most people to easily hit a stun, click the hotkey to switch to an emp and disable anything that could give what was happening away, then switch back to refresh the stun and do the deed

**Comment by brndd** [comment]
>Player behavior and competence isnt exactly a great argument for the strength of an item.

Maybe not, but it's a great argument for whether the strength of an item is a problem or not. Spess isn't an esport. Things don't need to be "balanced" beyond removing things that actively make the game less fun for everyone.

**Comment by ghost** [comment]
An actual counter to SPS? Merged never.

**Comment by Blithering** [comment]
Whats the capacity like in comparison to the rifle? If its similar I'd argue to put the rifle in the lathe and have this as the one that spawns in the ops shuttle.

**Comment by aceedex** [comment]
and Ill argue that something that can disable radio, suit sensors, S & GPS, force open doors, drain the more common on-station weapon types, at range, being mass produce able isnt exactly fun or interactive or makes rounds better or what have you.

Another thing that REALLY bothers me about this addition is that it severely FUCKS with mechs. ever since #22575 mechs are (thankfully) very much susceptible to ions. But its to my understanding that that has always worked without pushing mechs into irrelevancy because of the relative rarity of EMPs

**Comment by SonixApache** [comment]
Pistol: 5 shots, weak EMP, fits in holsters and backpacks and on your belt
Carbine: 7 shots, weak EMP, fits in backpacks and on your belt
Rifle: 10 shots, 'strong' EMP, fits on your back

Again, I'd like to have this as a 'high tech research' type of deal.

**Comment by aceedex** [comment]
the sad part of the situation is that if it aint Power 5, plasma 4, NT 1 or anomaly anything it aint really high tech research

**Comment by brndd** [comment]
> and Ill argue that something that can disable radio, suit sensors, S & GPS, force open doors, drain the more common on-station weapon types, at range, being mass produce able isnt exactly fun or interactive or makes rounds better or what have you.

Sure, they can be powerful in some situations, but I just don't agree that they have much round gunking potential overall. If it turns out I'm wrong, it's a 12 line change to remove them from the protolathe.

**Comment by jknpj** [comment]
Why not make these go to the armory and make the rifles printable?

**Comment by SonixApache** [comment]
Why would I do that?

**Pull request merged**